lib/logstash/outputs/influxdb.rb

```diff
@@ -57,7 +60,7 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   # Set the level of precision of `time`
   #
   # only useful when overriding the time value
-  config :time_precision, :validate => ["m", "s", "u"], :default => "s"
+  config :time_precision, :validate => ["n", "u", "ms", "s", "m", "h"], :default => "ms"
```

Comment:
Correct. I'm aligning this plugin with Influxdb 0.9 — which now supports the above time precisions — and setting the plugin's default value to match Influxdb's. While the plugin's default doesn't _have_ to change to match Influxdb's default, it seems more predictable to have them do so.